dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -406,8 +488,19 @@ def prepare_airflow_packages(
             version_suffix_for_pypi=version_suffix_for_pypi,
         )
     get_console().print("[success]Successfully prepared Airflow packages:")
-    for file in sorted(DIST_DIR.glob("apache_airflow*")):
-        get_console().print(file.name)
+    packages = tuple(_dist_packages(package_format=package_format, build_type="airflow"))
+    for dist_info in packages:
+        get_console().print(str(dist_info))
+    get_console().print()
+    if package_format == "wheel":
+        return
+
+    for dist_info in packages:
+        if dist_info.dist_type == "sdist":
+            get_console().print(
+                f"[info]Validate build wheel from sdist distribution: {dist_info.filepath.name!r}[/]"
+            )
+            _check_sdist_to_wheel(dist_info)
```

Comment:
This check run pretty fast, for about 5-15 seconds, but I've also have a filling that this check might be optional

Reply:
It's fine - let's keep it. The more guardrails we have while releasing packages, the better.